src/Handlers/PaymentResponseHandler.php

```diff
@@ -217,7 +217,7 @@ public function handleShopwareApis(
         }
 
         // Only store additional data for the transaction if this is the first additional data
-        $additionalData = $this->paymentResponseHandlerResult->getAction();
+        $additionalData = $this->paymentResponseHandlerResult->getAdditionalData();
```

Comment:
Why were we getting the action in the first place?

Reply:
it was a typo, we get action a few lines above

Reply:
pretty sure by mistake. The variable name also says additionalData :/